template.rb

```diff
@@ -34,7 +34,10 @@ def rails_6_or_newer?
 end
 
 def add_gems
-  gem 'cssbundling-rails'
+  unless IO.read("Gemfile") =~ /^\s*gem ['"]cssbundling-rails['"]/
```

Comment:
I followed the same pattern as https://github.com/mattbrictson/rails-template/blob/8f7628fb98c46c1a78113c58832cc4e67ff031ab/template.rb#L151

Reply:
Awesome, that should work nicely. 👍